src/pdn/src/PdnGen.tcl

```diff
@@ -3408,11 +3408,13 @@ proc generate_lower_metal_followpin_rails {} {
       set vdd_name [get_voltage_domain_power [get_voltage_domain $xMin [expr $vdd_y - $width / 2] $xMax [expr $vdd_y + $width / 2]]]
       set vss_box [::odb::newSetFromRect $xMin [expr $vss_y - $width / 2] $xMax [expr $vss_y + $width / 2]]
       set vss_name [get_voltage_domain_ground [get_voltage_domain $xMin [expr $vss_y - $width / 2] $xMax [expr $vss_y + $width / 2]]]
+      # generate power_rails using first domain_power
+      set first_power_name [lindex $vdd_name 0]
```

Comment:
The primary_power of a voltage domain should be restricted to the stdcell rail power for that domain. If there are more power/ground nets then those should be considered "additional" rather than "primary"